src/applications/personalization/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -77,14 +102,16 @@ class RatedDisabilityList extends React.Component {
     if (!this.props.ratedDisabilities) {
       return <LoadingIndicator message="Loading your information..." />;
     }
-    // Display error message based on error type.
     if (
-      this.props.ratedDisabilities.errors ||
+      this.props?.ratedDisabilities?.errors ||
       this.props?.ratedDisabilities?.ratedDisabilities.length === 0
     ) {
+      // There are instances when a 200 response is received but evss sends an empty array.
+      // In this scenario errorCode is explicitly set to 404 to ensure a defined value is passed to noDisabilityRatingContent
+      const errorCode = this.props?.ratedDisabilities?.errors?.[0]?.code || 404;
```

Comment:
Turns out this line needed to be fixed anyway 😬 . I decided to explicitly assign value to errorCode, in the event the left hand expression turns out to be undefined.